### ISSUE TITLE
ValidatorFragment2 generates an overridden version of addComboBox for Xbase languages

### ISSUE THREAD

**Issue by LorenzoBettini** [eclipse/xtext-core#771]
Xbase already defines this method 

```
org.eclipse.xtext.xbase.ui.validation.XbaseValidationConfigurationBlock.addComboBox(String, String, Composite, int)
```

but the configuration block generated for Xbase languages overrides this method, just like for any other languages (indeed there's a warning in the generated configuration block, though src-gen is usually configured to exclude additional warnings).

Is this the intended behavior? Shouldn't Xbase languages reuse the one from XbaseValidationConfigurationBlock? @holgerschill

**Comment by cdietrich** [comment]
for me this looks like a oversight.

**Comment by cdietrich** [comment]
@holgerschill  any news on this issue?

**Comment by cdietrich** [comment]
ping @holgerschill

**Comment by holgerschill** [comment]
Fixed in 2.15
